packages/e2e/features/ui/components/authenticator/verify-user.feature

```diff
@@ -7,14 +7,15 @@ Feature: Verify User
   Background:
     Given I'm running the example "ui/components/authenticator/sign-in-with-email"
 
-  @angular @next @vue
```

Comment:
@slaymance just merged #369 that fixes account verification -- can you try adding these again?

Reply:
I resolved the conflicts, but perhaps incorrectly with @6b4ebc.

Reply:
Looks like you fixed it! Thanks!